labs/litellm_service/local.py

```diff
@@ -0,0 +1,28 @@
+from ollama import Client
+
+
+class RequestLocalLLM:
+    def __init__(self):
+        self.client = Client(host="http://localhost:11434")
```

Comment:
Should this be hardcoded?

Reply:
True, that should come from an env var, I'll add it